.changeset/big-fishes-rest.md

```diff
@@ -0,0 +1,5 @@
+---
+'@launchpad-ui/core': patch
+---
+
+Adding build:watch tooling to enable local linking
```

Comment:
You can delete this changeset file - since nothing new is being shipped to consumers we don't need to bump versions

Reply:
Ah, thanks! I thought it was required but that makes sense.